sdk/core/Azure.Core.TestFramework/src/TestRecording.cs

```diff
@@ -199,6 +200,42 @@ public TestRandom Random
 
         public string RecordingId { get; private set; }
 
+        private bool _useDefaultClientRequestIdFormat;
+
+        /// <summary>
+        /// Retrieves the value for the enviroment variable RECORDING_DEFAULT_ClIENT_GUID
+        /// </summary>
+        public bool UseDefaultClientRequestIdFormat
```

Comment:
This should be set based on RecordedTestBase, not based on the Variables.

Reply:
Changed to just pull from RecordedTestBase